### DISCUSSION TITLE
[feature request] Plans for operating systems support outside of the currently supported ones

### DISCUSSION THREAD

**Discussion by ghost** [ghostty-org/ghostty#3905] [Feature Requests, Ideas] [answered]
Hi

i have recently seen this project and it looks amazing.
as it however currently supports only macos and linux and the unix like front i cant really use it,
and since i dont know zig i cant port it.
i am using freebsd and would like to know if there is any interest on the sides of the dev team to support
operating systems such as freebsd, openbsd and netbsd

i understand that that is a big ask and hope it is not to big of a bother to ask

i wish you luck with the project

**Comment by JakeWharton** [comment, marked as answer]
See https://github.com/ghostty-org/ghostty/discussions/3832#discussioncomment-11688382. If you want to contribute patches to help it work that would be welcome.

**Reply by ghost** [reply]
as i said above i dont know zig and it is not that high on my "languages i want to learn list".
maybe i can find people who know zig which also know enough about bsd to help patches.
this terminal emulator does really look like what i waited for in terms of terminals lol

**Reply by JakeWharton** [reply]
Ah, sorry. I glossed over that part! If that's the case then I'm not sure you have much recourse except waiting for someone to pick it up.

**Reply by ghost** [reply]
i am saddly aware but knowing that the team is open to it is already quite neat
i think we have a list of software people want ported i may try to add ghostty to it

**Reply by ghost** [reply]
i think i should close this as most of the discussion is going on in the post you linked
